introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati nisi rem est error iste voluptas dolor in. Facilis, incidunt, soluta. Accusamus suscipit eum neque, hic. Deleniti blanditiis earum fuga ab.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.